core/primitives/src/version.rs

```diff
@@ -8,6 +8,7 @@ use crate::types::Balance;
 pub struct Version {
     pub version: String,
     pub build: String,
+    pub rustc_version: String,
```

Comment:
I always wonder if it is correct to add fields to `derive(Deserialize)` structs. If this is used by, eg, indexer, then the indexer would fail to deserialize responses from older nodes.